### PR TITLE
Add nr1-victory-visualizations listing

### PR DESCRIPTION
**Pull request by jerelmiller**
Adds the [nr1-victory-visualizations](https://github.com/newrelic/nr1-victory-visualizations) project to the OSS site

### CODE REVIEW

**Comment by jerelmiller** [comment]
Hold on merging until we go public with the repo. I can come back to merge this PR after its approved and the repo has been made public.

**Review by zstix — DISMISSED** [review]

**Comment by jerelmiller** [comment]
We just made that repo public, so this PR is good to go. @zstix looks like the titles are supposed to be title case, so I've updated this PR. Would you mind giving a quick re-review?

**Review by zstix — APPROVED** [review]

**Pull request merged**
